src/Servers/Kestrel/Core/src/Internal/Http/HttpResponseTrailers.cs

```diff
@@ -5,11 +5,22 @@
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http
 {
     public partial class HttpResponseTrailers : HttpHeaders
     {
+        private static long ParseContentLength(string value)
```

Comment:
Yeah, this is weird. Trailers aren't even allowed to have a content-length header.

Reply:
Removed